packages/quantic/force-app/main/default/lwc/quanticFeedbackModalQna/quanticFeedbackModalQna.js

```diff
@@ -0,0 +1,192 @@
+import documentUrl from '@salesforce/label/c.quantic_FeedbackDocumentUrl';
+import documentUrlExplanation from '@salesforce/label/c.quantic_FeedbackDocumentUrlExplanation';
+import helpUsImprove from '@salesforce/label/c.quantic_FeedbackHelpUsImprove';
+import answerNotSure from '@salesforce/label/c.quantic_FeedbackNotSure';
+import questionDocumented from '@salesforce/label/c.quantic_FeedbackQuestionDocumented';
+import questionHallucination from '@salesforce/label/c.quantic_FeedbackQuestionHallucination';
+import questionReadable from '@salesforce/label/c.quantic_FeedbackQuestionReadable';
+import questionTopic from '@salesforce/label/c.quantic_FeedbackQuestionTopic';
+import answerNo from '@salesforce/label/c.quantic_No';
+import sendFeedback from '@salesforce/label/c.quantic_SendFeedback';
+import skip from '@salesforce/label/c.quantic_Skip';
+import thankYouForYourFeedback from '@salesforce/label/c.quantic_ThankYouForYourFeedback';
+import answerYes from '@salesforce/label/c.quantic_Yes';
+import yourFeedbackHelps from '@salesforce/label/c.quantic_YourFeedbackHelps';
+import requiredFeedbackQuestion from '@salesforce/label/c.quantic_requiredFeedbackQuestion';
+import LightningModal from 'lightning/modal';
+import {api} from 'lwc';
+import feedbackFormTemplate from './quanticFeedbackModalQna.html';
+import successTemplate from './success.html';
+
+/** @typedef {"yes"|"unknown"|"no"} GeneratedAnswerFeedbackOption */
+
+/**
+ * The `QuanticFeedbackModalQna` component overlays a message modal on top of the current app window, the modal contains a form that allows the user to give feedback on a CRGA response.
+ *
+ * Under the hood, the component relies on a [`lightningModal`](https://developer.salesforce.com/docs/component-library/bundle/lightning-modal/documentation) component.
+ * For an example of how to use the `QuanticFeedbackModal` component, see the [`quanticSmartSnippet`](https://github.com/coveo/ui-kit/blob/master/packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/quanticSmartSnippet.js) implementation.
+ *
+ * @category Search
+ * @category Insight Panel
+ */
+export default class QuanticFeedbackModalQna extends LightningModal {
+  labels = {
+    skip,
+    done,
+    sendFeedback,
+    helpUsImprove,
+    requiredFeedbackQuestion,
+    thankYouForYourFeedback,
+    answerEvaluation,
+    questionTopic,
+    questionHallucination,
+    questionDocumented,
+    questionReadable,
+    yourFeedbackHelps,
+    answerYes,
+    answerNo,
+    answerNotSure,
+    documentUrl,
+    documentUrlExplanation,
+    additionalNotes,
+    additionalNotesExplanation,
+  };
+
+  /**
+   * @api
+   * The function that will be executed when the feedback is submitted.
+   * @type {function}
+   */
+  @api handleSubmit;
+
+  /**
+   * @api
+   * If the feedback modal was opened on the thumbs up. (otherwise it was on a thumbs down).
+   * @type {boolean}
+   */
+  @api liked;
```

Comment:
```suggestion
  /**
   * @api
   * The function that will be executed when the feedback is submitted.
   * @type {function}
   */
  @api handleSubmit;
  /**
   * @api
   * If the feedback modal was opened on the thumbs up. (otherwise it was on a thumbs down).
   * @type {boolean}
   */
  @api liked;
```